openslides/urls.py

```diff
@@ -11,14 +11,15 @@
 urlpatterns = patterns(
     '',
     url(r'^rest/', include(router.urls)),
-    # TODO: add "special" urls, for example pdf views etc.
+    (r'^users/', include('openslides.users.urls')),
```

Comment:
I would say this is just old style and we should use url-style everywhere. See line 13.